lib/mariaex/geometry.ex

```diff
@@ -0,0 +1,23 @@
+if Code.ensure_loaded?(Ecto.Type) do
```

Comment:
That would go in another library that a user of Ecto would put in their deps, perhaps Geo.